parallel_benchmark_test.go

```diff
@@ -0,0 +1,225 @@
+package roaring
+
+import (
+	"math/rand"
+	"runtime"
+	"testing"
+)
+
+type parOp int
+
+const (
+	parOpAnd parOp = iota
+	parOpOr
+	parOpXor
+	parOpAndNot
+)
+
+var defaultWorkerCount int = runtime.NumCPU()
+
+var GlobalParAggregator = NewParAggregator()
```

Comment:
I am a bit uneasy about this global aggregator. This needs to receive due consideration. My preference would be to minimize global variables.